test/link-fragments.md

```diff
@@ -125,6 +127,10 @@ Text
 
 <A href="https://example.com" id="HREFandID">Text</A>
 
+<p id="paragraph-id"></p>
+
+<p name="not-an-anchor"></p>
```

Comment:
Please add another case that has p/name to confirm that is NOT allowed.

Reply:
Sorry, I don't understand. This case is checked on line 148 to confirm that the `name` tag is not accepted on a non-`a` element. Had you missed that, or are you asking for a different case?

By the way, I changed the names to make it more obvious what is being tested.